examples/hello/package.json

```diff
@@ -3,7 +3,10 @@
   "description": "Tabris.js - Hello, World!",
   "category": "Small Apps",
   "main": "hello.js",
+  "scripts": {
+    "start": "http-server -c-1 -d false"
```

Comment:
For 2.0, we recommend the CLI over starting a plain HTTP server: `tabris serve`. This should make the npm script redundant.